routes/signup.js

```diff
@@ -1,9 +1,8 @@
-var httpRequest = require('request');
+var hatchet = require('hatchet');
+var url = process.env.SIGNUP;
 
-module.exports = function(transaction, callback) {
-  var url = process.env.SIGNUP;
-  callback = callback || function() {};
-  httpRequest.post({
+module.exports = function(transaction) {
+  hatchet.send("sign_up_for_mofo_newsletter", {
     url: url,
```

Comment:
If you make the sawmill worker more generic, you'll need to add the `method: 'POST'` key here